source/_static/js/instantSearch.js

```diff
@@ -382,6 +381,8 @@ window.addEventListener("DOMContentLoaded", () => {
             platform = activePlatform;
           }
 
+          var pathname = environment === "dev" ? "/search.html" : `/docs/minio/${platform}/search.html`;
```

Comment:
Another concern here is that @ravindk89 is in the process of adding a bunch of additional k8s platforms, which means the 377-383 `if` block is going to have to greatly expand. And each time we add a new k8s platform, we'll have to remember to add it to this.

Reply:
might not be another way around that for now. Thanks for the reminder as well.

Reply:
Not sure how to handle this the right way. Do you think this can be handled programatically by defining both the main platform and sub somewhere in the `default.conf.py`?